src/openrct2/ride/coaster/meta/ClassicWoodenRollerCoaster.h

```diff
@@ -35,7 +35,7 @@ constexpr const RideTypeDescriptor ClassicWoodenRollerCoasterRTD =
    SET_FIELD(NameConvention, { RideComponentType::Train, RideComponentType::Track, RideComponentType::Station }),
    SET_FIELD(EnumName, nameof(RIDE_TYPE_CLASSIC_WOODEN_ROLLER_COASTER)),
    SET_FIELD(AvailableBreakdowns, (1 << BREAKDOWN_SAFETY_CUT_OUT) | (1 << BREAKDOWN_RESTRAINTS_STUCK_CLOSED) | (1 << BREAKDOWN_RESTRAINTS_STUCK_OPEN) | (1 << BREAKDOWN_VEHICLE_MALFUNCTION) | (1 << BREAKDOWN_BRAKES_FAILURE)),
-   SET_FIELD(Heights, { 41, 24, 8, 11, }),
+   SET_FIELD(Heights, { 24, 24, 8, 11, }),
```

Comment:
Since this is now lower and by quite a bit, players will have less freedom on how high to build? A track design that could be placed before, now wouldn't be possible to be changed at the highest points.

Reply:
This is the original height the classic wooden coaster should have as it was in RCT1 and keep in mind this ride type is meant to be a port of the RCT1 wooden coaster to openrct2, the height it currently has is an unintentional leftover from when it was being created from the RCT2 wooden coaster. 
Any designs made with RCT1 itself should still all be placeable, this would only affect classic wooden coaster designs made in openRCT2 between now and the ride type being introduced in last october.
Other similar changes have been made to it and the similarly ported classic mini coaster since their reintroductions in openrct2 to correct other accuracy differences that still would have affected track designs made with openrct2, notably the max lift speed being lowered on the classic woodie recently, and the ride ratings being change for both of them to better reflect their RCT1 values.